worker.go

```diff
@@ -64,12 +65,30 @@ func initWorkers(opt []workerOpt) error {
 	return nil
 }
 
+func getWorkerKey(name string, filename string) string {
+	key := filename
+	if strings.HasPrefix(name, "m#") {
```

Comment:
Couldn't we always use the name if the filename is duplicated? This would remove a soft coupling with the Caddy module.

Reply:
We wanted to error out if the filename of global workers was duplicated, hence we'd still need a prefix check somewhere. I might be mistaken, but I don't see a way around it without infringing on BC in some way.

Reply:
It's not a BC break, but it's mixing the responsibilities of the standalone library and of the Caddy module. The notion of "module" is Caddy-specific and should leak into the standalone lib.